### ISSUE TITLE
pip install fails on python 2

### ISSUE THREAD

**Issue by cosama** [lbl-anp/becquerel#165]
If I try to install `becquerel` with python2.7 it fails with
```
Collecting becquerel
  Using cached https://files.pythonhosted.org/packages/ca/9f/cc7dda7f6d0092315e684e26c03350933c79a7114f82344a6237534e4908/becquerel-0.2.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-HwAVfc/becquerel/setup.py", line 37, in <module>
        with open('CONTRIBUTING.md', 'r') as fh:
    IOError: [Errno 2] No such file or directory: 'CONTRIBUTING.md'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-HwAVfc/becquerel/
```
It works with python3.6. In the archive that is linked the CONTRIBUTING.md file is indeed missing.

**Comment by cosama** [comment]
I guess we need to add a `MANIFEST.in` file to add `CONTRIBUTING.md`, `RELEASING.md`, `LICENCE.txt`, `requirements.txt` and `requirements-dev.txt` (might not be absolutely necessary) as those files are used in the `setup.py` file.

Also I saw that the License is marked as `UNKNOWN` in the `PKG-INFO` file in that archive. Something we should fix too.

**Comment by cosama** [comment]
Adding `license=LICENSE,` to the `setup()` call in `setup.py` seems to fix the license tag issue in `PKG-INFO`.

I think we have to fix these two things and go for a batch. Do we have a procedure for that? Where do we branch off, `develop` or `master`?

**Comment by markbandstra** [comment]
I think we should separate the license issue from the installation issue.

Regarding the license itself, I removed `license=LICENSE` from the `setup()` call in d8e36ee9cb because when testing with TestPyPI I found that the long text of our license was not compatible with the PyPI package page formatting. Basically it is expecting a short, single-line string like `"OSI Approved (BSD)"`, etc.

**Comment by jccurtis** [comment]
Thanks @cosama and @markbandstra!

Lets make a quick patch (`0.2.1`) with the following:

- Add the `--universal` option to the `sdist` build as [here](https://stackoverflow.com/questions/30438216/how-do-i-upload-a-universal-python-wheel-for-python-2-and-3)
- Add a `MANIFEST`. See the [docs](https://docs.python.org/2/distutils/sourcedist.html#manifest-template)
- Include a fix for #166

**Comment by jccurtis** [comment]
@markbandstra I'm taking a stab at this now ...

**Comment by markbandstra** [comment]
Closed by #168
